Jenkinsfile

```diff
@@ -562,10 +562,43 @@ pipeline {
           }
         }
 
+	stage('ROS MC mission box (LPE)') {
+          agent {
+            docker {
+              image 'px4io/px4-dev-ros:2018-06-19'
+              args '-e CCACHE_BASEDIR=$WORKSPACE -v ${CCACHE_DIR}:${CCACHE_DIR}:rw -e HOME=$WORKSPACE'
+            }
+          }
+          options {
+            skipDefaultCheckout()
+          }
+          steps {
+            sh 'export'
+            sh 'rm -rf build; rm -rf px4-posix_sitl_default*; rm -rf .ros; rm -rf .gazebo'
+            unstash 'posix_sitl_package'
+            sh 'tar -xjpvf build/posix_sitl_default/px4-posix_sitl_default*.bz2'
+            sh 'px4-posix_sitl_default*/px4/test/rostest_px4_run.sh mavros_posix_test_mission.test mission:=multirotor_box vehicle:=iris est:=lpe'
+          }
+          post {
+            always {
+              sh 'px4-posix_sitl_default*/px4/Tools/upload_log.py -q --description "${JOB_NAME}: ${STAGE_NAME}" --feedback "${JOB_NAME} ${CHANGE_TITLE} ${CHANGE_URL}" --source CI .ros/rootfs/fs/microsd/log/*/*.ulg'
+              archiveArtifacts '.ros/**/*.pdf'
+              archiveArtifacts '.ros/**/*.csv'
```

Comment:
remove if not processing estimator data

Reply:
@lamping7 these are outputs from the ekf log data processing or `upload_log.py`?